README.md

```diff
@@ -110,10 +104,6 @@ make tc
 make pc
 ```
 
-- Update the cookiecutter template
-
-```sh
-make u
```

Comment:
why remove this?

Reply:
Updating the template might not be required for this project once itis created, right?